src/pages/page-2.js

```diff
@@ -3,13 +3,160 @@ import { Link } from "gatsby"
 
 import Layout from "../components/layout"
 import Seo from "../components/seo"
+import imgLogo from "../images/Luma.png"
+import imgCart from "../images/Cart.png"
 
 const SecondPage = () => (
   <Layout>
     <Seo title="Page two" />
-    <h1>Hi from the second page</h1>
-    <p>Welcome to page 2</p>
+    {/* <h1>Hi from the second page</h1>
+    <p>Welcome to page 2</p> */}
     <Link to="/">Go back to the homepage</Link>
+    <h1></h1>
+
+    <div className="flex flex-row">
+      <div className="divHeaderCell w-8/12">
+        <img src={imgLogo} />
+      </div>
+      <div className="divHeaderCell w-3/12">
+        <input type="text" placeholder="Search entire store here..." />
+      </div>
+      <div className="divHeaderCell w-1/12">
+        <img src={imgCart} />
+      </div>
+    </div>
+
+    <div className="flex py-0 font-bold bg-gray-200 invisible md:px-2 md:visible md:py-2">
+      <label>Category</label>
```

Comment:
Improper use of `<label>`  you want to describe a headline.

Reply:
Called a heading. I created a style for <h2> and used that for all my section headings. However, would it be better to use <h1> and give them each a classname as I originally did with the labels?

Reply:
can you give an example?

Reply:
`h2 { @apply text-sm text-gray-400 uppercase mb-2 }<h2>Contact Information</h2>`

Reply:
```
h2 { @apply text-sm text-gray-400 uppercase mb-2 }
<h2>Contact Information</h2>

 - or - 

.sectionHeader { @apply text-sm text-gray-400 uppercase mb-2 }
<h1 className="sectionHeader">Contact Information</h1>
```

Reply:
either is fine just be consistent and explain why you chose what you chose if someone asks. Ill leave it to your discretion.

Reply:
Went with latter version as seemed less prone to issues if other "h2"s are used elsewhere